package exceptionHandling;

public class tryCatch {
	
	public static void main(String[] args) {
		int ar[] =  {23, 345, 56, 55};
		
		try {
		System.out.println(ar[6]);
		}
		catch(Exception e) {
			e.printStackTrace();
		}
		
	}
	
}
